spiceaidocs/docs/components/data-connectors/index.md

```diff
@@ -11,24 +11,97 @@ Data Connectors provide connections to databases, data warehouses, and data lake
+| `file`          | File        | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
+| `flightsql`     | FlightSQL   | Alpha  | Arrow Flight SQL                    | `full`           | ❌               | ❌                |
+| `ftp`, `sftp`   | FTP/SFTP    | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
+| `graphql`       | GraphQL     | Alpha  | GraphQL                             | `full`           | ❌               | ❌                |
+| `github`        | GitHub      | Alpha  | GraphQL, REST                       | `full`           | ❌               | ❌                |
+| `http`, `https` | HTTP(s)     | Alpha  | Parquet, CSV                        | `full`           | ❌               | ❌                |
+| `mysql`         | MySQL       | Alpha  |                                     | `full`           | ❌               | ❌                |
+| `odbc`          | ODBC        | Alpha  | ODBC                                | `full`           | ❌               | ❌                |
+| `postgres`      | PostgreSQL  | Alpha  |                                     | `full`           | ❌               | ❌                |
+| `snowflake`     | Snowflake   | Alpha  | Arrow                               | `full`           | ❌               | ❌                |
+| `spiceai`       | Spice.ai    | Alpha  | Arrow Flight                        | `append`, `full` | ✅               | ❌                |
+| `s3`            | S3          | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
+| `sharepoint`    | SharePoint  | Alpha  |                                     | `full`           | ❌               | ✅                |
+| `spark`         | Spark       | Alpha  | Spark Connect                       | `full`           | ❌               | ❌                |
+
+## Object Store File Formats
+For data connectors that are object store compatible, if a folder is provided, the file format must be specified with `params.file_format`.
+
+If a file is provided, the file format will be inferred, and `params.file_format` is unnecessary.
+
+File formats currently supported are:
+
+| Name                                          | Parameter               | Supported | Is Document Format |
+| --------------------------------------------- | ----------------------- | --------- | ------------------ |
+| [Apache Parquet](https://parquet.apache.org/) | `file_format: parquet`  | ✅        | ❌                |
+| [CSV](/reference/file_format.md#csv)          | `file_format: csv`      | ✅        | ❌                |
+| [Apache Iceberg](https://iceberg.apache.org/) | `file_format: iceberg`  | Roadmap   | ❌                |
+| JSON                                          | `file_format: json`     | Roadmap   | ❌                |
```

Comment:
Several formats are marked as Roadmap where we have example for them, for example Markdown below.